reviews/graduation-jaeger.md

```diff
@@ -0,0 +1,156 @@
+  * [Weaveworks](https://www.weave.works/) empowers developers and DevOps teams to build better software faster. Using Jaeger in production since Spring 2018. ~70 microservices, ~600 containers, most of services are instrumented with Jaeger. Using AWS Elasticsearch as the span storage backend, with two weeks retention. 
+    * Bryan Boreham at KubeCon EU 2018. [How we used Jaeger and Prometheus to deliver lightning-fast user queries](https://www.youtube.com/watch?v=qg0ENOdP1Lo).
+  * [ContaAzul](http://contaazul.com), ERP for small and medium business in Brazil.
+    * One legacy monolith with ~60 microservices around it, 20 of them already instrumented and the rest being prioritized. Microservices are deployed to a self-maintained Kubernetes cluster on AWS. Each Kubernetes node holds a Jaeger agent responsible for collecting spans from its node pods. Span storage is on AWS Elasticsearch Service; it currently has 2B searchable documents related to Jaeger spans.
+    * Quote: "Jaeger is helping us to increase observability since Feb 2018 and is praised by our developers."
+  * [Weave HQ](https://www.getweave.com/) threads together data, software and communication platforms to build stronger relationships at the point of contact. Deployed at over 8000 medical offices. Using Jaeger since early 2018; ~150 active microservices, ~500 containers, ~90% of microservices are instrumented for tracing. Using Cassandra as span storage backend, 1 week retention, ~2M spans/day.
+    * Fun fact 1: started collecting fewer metrics in Prometheus because Jaeger works much better for root cause analysis than having extra metrics.
+    * Fun fact 2: most application logs are written to Jaeger spans via the OpenTracing API and viewed in Jaeger UI in the context of the trace, instead of log aggregation tools like ELK.
+  * [Candide](https://candide.eu/), a mobile app for gardeners, uses Jaeger in production running on GKE with an Elasticsearch backend hosted on Elastic Cloud. 30+ microservices.
+  * [Ticketmaster](https://www.ticketmaster.com/), a ticket sales and distribution company based in Beverly Hills, California, with operations around the world.
+    * Traces 100 million transactions per day with Jaeger .
+    * Has 50-plus services instrumented and nine groups of engineering teams actively using Jaeger.
+    * Quote: "Jaeger Tracing is helping us achieve our vision of observability capabilities across different versions of our platforms, how they integrate with each other and how they have grown over time." -- Kraig Amador, Senior Director.
+  * [Red Hat](https://www.redhat.com/en), world’s leading provider of enterprise open source solutions, using a community-powered approach to deliver high-performing Linux, cloud, container, and Kubernetes technologies.
+    * Jaeger will be fully supported by Red Hat as part of its OpenShift Service Mesh product due for release in a couple of months.
```

Comment:
Any status update on this one?

Reply:
Jaeger is part of the Red Hat OpenShift Service Mesh offering as of a couple of weeks ago:

> Based on the Istio, Kiali and Jaeger projects and enhanced with Kubernetes Operators, OpenShift Service Mesh is designed to deliver a more efficient, end-to-end developer experience around microservices-based application architectures. This helps to free developer teams from the complex tasks of having to implement bespoke networking services for their applications and business logic.

https://www.redhat.com/en/about/press-releases/red-hat-launches-openshift-service-mesh-accelerate-adoption-microservices-and-cloud-native-applications

Reply:
added